src/gsi/read_nsstbufr.f90

```diff
@@ -260,6 +279,31 @@ subroutine read_nsstbufr(nread,ndata,nodata,gstime,infile,obstype,lunout, &
            else
               sst = bmiss
            endif
+        elseif ( trim(subset) == 'NC031005' ) then            ! Argo or Glider
+           msst = 12.0_r_kind                                 ! for ARGO, assign to be 12
+           call ufbint(lunin,tpf3,3,65535,klev,'SSTH SALNH WPRES') ! get klev,T,water pres & salinity
+
+           call ufbint(lunin,r_rpid,1,1,iret,'WMOP')
+           write(crpid,'(I7)') int(r_rpid)
+
+           apres(:) = real(tpf3(3,:))
+
+           pres1 = minval(apres)/10000.0_r_kind   ! converting wpres from Pa to dbar for minmum depth 
+                                                  ! Not always layer 1 
+!
+!          get zob in meters
+!
+           zob = ((-3.434e-12_r_kind*pres1+1.113e-7_r_kind)*pres1+0.712953_r_kind)*pres1 + 14190.7_r_kind*log(one+1.83e-5_r_kind*pres1)
```

Comment:
The coding standards call for no more than 132 characters per line.  Line 296 has 135 characters.  Please break up this line to adhere to the 132 character limit.

Reply:
@MichaelLueken-NOAA : Done